### ISSUE TITLE
Modify rules for handling indentation of closing parentheses

### ISSUE THREAD

**Issue by w0rp** [eslint/eslint#7522]
* **ESLint Version: 3.5.0**
* **Node Version: 6.4.0**
* **npm Version: 3.10.7**

eslint has rules for checking the indentation of code, but does not handle closing parentheses, so the following code is valid with all configurations:

```javascript
foo(
  )
```

This can cause problems when you re-indent code. For example. I recently changed my codebase from 4 to 2 space indentation, and used `--fix` to fix the indentation my code. The end result is that code that looked like this.

```javascript
foo(
    bar(
    )
)
```

Now looks like this:

```javascript
foo(
  bar(
    )
)
```

Would it be possible to modify the `indent` rule to include an option for closing parentheses?

**Comment by not-an-aardvark** [comment]
Thanks for this report. This is actually a bug in the `indent` rule; it should be checking closing parens.

You might also want to follow https://github.com/eslint/eslint/issues/7274.

**Comment by not-an-aardvark** [comment]
Actually, after looking over the other `indent` issues, I think this issue is a duplicate of https://github.com/eslint/eslint/issues/1801 and https://github.com/eslint/eslint/issues/6007. Do you agree?

**Comment by w0rp** [comment]
I'm not sure. One is about braces, the other is about Arrays. I think all of them are kind of similar. Either way, I think all of these issues should be taken into consideration when the fixes are made.
